### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by jcaraballo17**

### CODE REVIEW

File: src/WebSDL/settings/base.py [hunk 1]
```diff
@@ -152,11 +154,14 @@
 
 RECAPTCHA_VERIFY_URL = "https://www.google.com/recaptcha/api/siteverify"
 
+EMAIL_SENDER = data['password_email_sender'] if 'password_email_sender' in data else '',
 
-EMAIL_SENDER = data['email_sender'] if 'email_sender' in data else '',
+NOTIFY_EMAIL = data['notify_email_sender'] if 'notify_email_sender' in data else ''
```
Comment (fryarludwig) [inline src/WebSDL/settings/base.py]: In the settings file, we call it 'notify_email_sender'. It's only semantic, but we should change it to 'no_reply_email_sender' or something of that ilk.

File: src/WebSDL/settings/settings_template.json [hunk 2]
```diff
@@ -3,8 +3,11 @@
   "host": "-optional",
   "recaptcha_secret_key": "-optional",
   "recaptcha_user_key": "-optional",
-  "email_sender": "password.reset@data.envirodiy.org",
+  "password_email_sender": "password.reset@data.envirodiy.org",
```
Comment (fryarludwig) [inline src/WebSDL/settings/settings_template.json]: Same thing as earlier - let's be clear we don't receive emails to this address - no_reply@data.envirodiy.or

File: src/dataloader/models.py [hunk 3]
```diff
@@ -728,7 +736,7 @@ class Variable(models.Model):
                                                   through='VariableExternalIdentifier')
 
     def __str__(self):
-        return '%s: %s' % (self.variable_name_id, self.variable_code)
+        return '%s' % self.variable_code
```
Comment (fryarludwig) [inline src/dataloader/models.py]: Styling only, but we could just do str(self.variable_code)?

File: src/dataloaderinterface/forms.py [hunk 4]
```diff
@@ -8,7 +10,14 @@
...
 from django.forms.formsets import formset_factory
 
-from dataloaderinterface.models import ODM2User
+from dataloaderinterface.models import ODM2User, SiteAlert
+
+
+class SiteTypeSelect(forms.Select):
+    def create_option(self, name, value, label, selected, index, subindex=None, attrs=None):
+        option = super(SiteTypeSelect, self).create_option(name, value, label, selected, index, subindex, attrs)
+        option['attrs']['title'] = self.choices.queryset.get(name=value).definition if value else ''
```
Comment (fryarludwig) [inline src/dataloaderinterface/forms.py]: Maybe a try/catch here, depending on the chance we get bad data. Not an issue if option['attrs']['title'] can receive an invalid or null value.

File: src/dataloaderinterface/management/commands/check_data_loss.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                       "data values in the last {} hours. The last update was on {}. You may want to check your "
+                       "equipment to ensure it's working as intended. \n\n"
+                       "https://data.envirodiy.org/sites/{}/\n\n"
+                       "Best regards,\n"
+                       "The EnviroDIY team.\n"
+                       "").format(site_alert.user.first_name, site_alert.site_registration.sampling_feature_name,
+                                  gap, site_alert.last_measurement_utc_datetime,
+                                  site_alert.site_registration.sampling_feature_code)
+            success = Command.send_email(site_alert.user.email, subject, message)
+            if success:
```
Comment (fryarludwig) [inline src/dataloaderinterface/management/commands/check_data_loss.py]: This won't fail unless there's a server issue. If the email is wrong, or the email sender (as we learned earlier) doesn't actually send the email, we won't know. We need to keep this here, but this check doesn't ensure the email was actually sent or received. This isn't something that can realistically be fixed, but it's good to be aware of.

File: src/dataloaderinterface/static/dataloaderinterface/js/site-form.js [hunk 6]
```diff
@@ -334,6 +344,17 @@ function initializeResultsTable() {
     bindResultDeleteEvent(rows);
 }
 
+function notifyInputStatus() {
```
Comment (fryarludwig) [inline src/dataloaderinterface/static/dataloaderinterface/js/site-form.js]: Love it.

File: src/WebSDL/settings/base.py (hunk 1 above)
Comment (jcaraballo17) [reply src/WebSDL/settings/base.py]: Noted. I'll add this to a list of things of issues to complete before the leaf pack network integration, in the meantime I'll change it in the server's settings.json file.

File: src/dataloaderinterface/forms.py (hunk 4 above)
Comment (jcaraballo17) [reply src/dataloaderinterface/forms.py]: I have the `... if value else ''` that puts an empty string in the title if there's no value. It won't crash, but I can do the try/catch to be 120% sure.

File: src/dataloaderinterface/management/commands/check_data_loss.py (hunk 5 above)
Comment (jcaraballo17) [reply src/dataloaderinterface/management/commands/check_data_loss.py]: Noted.

**Comment by SRGDamia1** [comment]
So, um.. when does it become live? :)

**Comment by horsburgh** [comment]
It's now live on the staging server - http://wpfwebstaging.uwrl.usu.edu/. Just waiting on Juan to give us the go-ahead to switch the IP addresses of the servers.

**Comment by jcaraballo17** [comment]
Never!

**Comment by jcaraballo17** [comment]
No, I'm joking. I was checking everything was okay. 
It all looks good, you can switch the IP addresses whenever you're ready now @horsburgh.

**Comment by horsburgh** [comment]
Bad timing - I'm running to teach my lab right this second. This will probably happen tomorrow.

**Comment by SRGDamia1** [comment]
Can you just ping me when the switch happens?  I have some variables that I'll put in right away.  :)

**Pull request merged**